insights/terraform/groups/main.tf

```diff
@@ -0,0 +1,59 @@
+locals {
+  # Add new group members here
+  member_map  = {
+    analysts  = ["HWRI", "SKKH"]
+    authors   = ["TUVE"]
+    readers   = []
+  }
+}
+
+# Add to the IAM groups
+
+data "aws_iam_group" "groups" {
```

Comment:
Shouldn't this pull from `aws_iam_user` or am I misunderstanding how this works?

Reply:
Nevermind, it took me slightly longer to reason that groups and group memberships were managed in separate places